### DISCUSSION TITLE
Idea Pile for the app

### DISCUSSION THREAD

**Discussion by irsaris** [uprm-inso4117-2023-2024-s2/semester-project-study-pet#22] [Ideas]
![image](https://github.com/uprm-inso4115-2023-2024-s2/semester-project-automated-resume-builder/assets/70551866/fab10e9c-64c4-49c7-8a45-40ac1a1a9c95)

![image](https://github.com/uprm-inso4115-2023-2024-s2/semester-project-automated-resume-builder/assets/70551866/673a0965-d5a5-4c23-89fd-aaedd89bf826)

![image](https://github.com/uprm-inso4117-2023-2024-s2/semester-project-study-pet/assets/70551866/3a30cdf5-32ed-42d6-bd01-351c70d38a26)

![image](https://github.com/uprm-inso4117-2023-2024-s2/semester-project-study-pet/assets/70551866/21b509f6-adaf-4c11-8509-2c29b8479716)

![image](https://github.com/uprm-inso4117-2023-2024-s2/semester-project-study-pet/assets/70551866/169c5294-10a8-42cc-843e-a8a7b3349e03)

**Comment by MarkoSchuetz** [comment]
Consider existing flash card apps to learn from their features. For example, TOFU learn, Anki, etc.

Also, what all are the activities for which the learner should be rewarded? If we imagine long-term learning we have several stages/activities.
1. Adding a deck to the set of personal decks to be learned.
2. Learning a new card from a deck until an initial level of recall has been achieved.
3. Reviewing formerly mastered cards.
4. Creating cards (on an imported deck or on a newly created deck).
5. Creating new deck.

For continued engagement it would not be enough to just work with one deck without creating new cards: eventually all cards will have been learned and the time between reviews of each card will be at the maximum. So, there should be an incentive to continue to learn. That could take the form of rewarding the addition of new cards/decks to the pool of cards to be learned. To grow the pool of all cards available as public decks rewards should be given for adding such decks or for adding cards to such decks.
